rundeck/rundeck
# RUN-2465 Feat/unit-test for activity list

**Is this a bugfix, or an enhancement? Please describe.**
This is an enhancement

**Describe the solution you've implemented**
Test cases:

- Verify the rendering of page information and total results.
- Check the display of current executions and finished jobs.
- Handle error messages when API calls fail.
- Display a message when no data is available.
- Trigger bulk actions like delete and search.
- Navigate to the execution page.
- Fetch data automatically and render messages for new executions since a timestamp.


**Describe alternatives you've considered**


**Additional context**
